test/integration/relay/connection.test.js

```diff
@@ -284,6 +331,112 @@ if (helper.sequelize.dialect.name === 'postgres') {
         );
       });
 
+      it('should not duplicate attributes', async function () {
+        let sqlSpy = sinon.spy();
+
+        let projectConnectionAttributesUnique;
+
+        const userProjectConnection = sequelizeConnection({
+          name: 'userProject',
+          nodeType: this.projectType,
+          target: this.User.Projects,
+          before(options){
+            // compare a uniq set of attributes against what is returned by the sequelizeConnection resolver
+            let getUnique = uniq(options.attributes);
+            projectConnectionAttributesUnique = getUnique.length === options.attributes.length;
+          },
+          orderBy: new GraphQLEnumType({
+            name: this.User.name + this.Project.name + 'ConnectionOrder',
+            values: {
+              ID: {value: [this.Project.primaryKeyAttribute, 'ASC']}
+            }
+          }),
+          edgeFields: {
+            isOwner: {
+              type: GraphQLBoolean,
+              resolve: function (edge) {
+                return edge.node.ownerId === edge.source.id;
+              }
+            }
+          }
+        });
+
+
+        const userType = new GraphQLObjectType({
+          name: this.User.name,
+          fields: {
+            ...attributeFields(this.User),
+            id: globalIdField(this.User.name),
+            tasks: {
+              type: this.userTaskConnection.connectionType,
+              args: {
+                ...this.userTaskConnection.connectionArgs,
+                completed: {
+                  type: GraphQLBoolean
+                }
```

Comment:
Isn't relevant either AFAIR (best to keep the test as small as possible while showing the issue)